proto/panacea/datadeal/v2/deal.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package panacea.datadeal.v2;
+
+option go_package = "github.com/medibloc/panacea-core/x/datadeal/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// Deal defines a deal.
+message Deal {
+  uint64 id = 1;
+  string address = 2;
+  repeated string data_schema = 3;
+  cosmos.base.v1beta1.Coin budget = 4;
+  uint64 max_num_data = 5;
+  uint64 cur_num_data = 6;
+  string consumer_address = 7;
+  DealStatus status = 8;
+  bytes nonce = 9;
```

Comment:
Do we still need this nonce? I think it was required to make consensus between oracles.

Reply:
The nonce is required for `AES256` encryption. It is used to store the nonce used when performing encryption/decryption with combineKey. In deal, it is used to encrypt and decrypt the `certificate` with combineKey. It seems that there will be no problem if this value is `Params`, but in the previous implementation, each deal had a separate nonce.

Reply:
And the CID of IPFS is always changed. Because encrypted data always changes when using a random nonce.

Reply:
> The nonce is required for AES256 encryption. It is used to store the nonce used when performing encryption/decryption with combineKey. In deal, it is used to encrypt and decrypt the certificate with combineKey. It seems that there will be no problem if this value is Params, but in the previous implementation, each deal had a separate nonce.

We can encrypt/decrypt data using `AES256` with randomly generated nonce.
Since the data is validated by the only one oracle, I think the nonce is not required.
In doracle, if we don't use nonce, all the oracles encrypt the data using each nonce, leading to each different value of `encrypted_oracle_priv_key` and consensus can't be made.
However in this scalable oracle, we don't need any consensus so that I thought the randomly generated nonce can be used. If I miss something, please let me know :pray:

Reply:
> And the CID of IPFS is always changed. Because encrypted data always changes when using a random nonce.

I don't understand the meaning of `the CID of IPFS is always changed`. Could you elaborate more??

Reply:
Yes, We can use a random nonce as you said.
However I'm worried that the data stored in IPFS keeps changing CID.(Even though it is the same data).

Reply:
Ah, I got your point.

Reply:
Right. As you said, we'd better to use nonce in encryption. Thank you all!

Reply:
Confirm. Remove the field 'nonce'.